level_complete.gd

```diff
@@ -0,0 +1,14 @@
+extends Area2D
+export var last_level = "res://Scenes/victory!.tscn"
```

Comment:
needs to be a preload

e.g. `preload("res://Scenes/victory!.tscn)`

Also, the tscn needs to be included AND should not have special characters in the name.